README.md

```diff
@@ -149,6 +149,14 @@ supported "system calls" are documented in the [`pyndustri.pyi`] file.
 To learn about the possible compiler errors, refer to the `ERR_` and `ERROR_DESCRIPTIONS`
 constants in the [`constants.py`] file.
 
+To install pyndustric as module run:
+```sh
+$ git clone https://github.com/Lonami/pyndustric.git
+$ pip install -e pyndustric
+```
+Alternativly, you can run it as module without installation by placing your program file at the root of the project directory.
```

Comment:
Just a tiny style preference of mine:

````suggestion
To install pyndustric as module run:

```sh
$ git clone https://github.com/Lonami/pyndustric.git
$ pip install -e pyndustric
```

Alternativly, you can run it as module without installation by placing your program file at the root of the project directory.
````